install-binary.sh

```diff
@@ -3,7 +3,7 @@
 # Shamelessly copied from https://github.com/technosophos/helm-template
 
 PROJECT_NAME="helm-diff"
-PROJECT_GH="databus23/$PROJECT_NAME"
+PROJECT_GH="philomory/$PROJECT_NAME"
```

Comment:
Perhaps this is your temporary change on the script for local testing, which isn't intended to be included in this PR?

Reply:
Oh, whoops, sorry, that wasn't supposed to be included in the PR, hahaha